teams/archive/interim-leadership-chat.toml

```diff
@@ -17,6 +18,7 @@ alumni = [
     "pietroalbini",
     "pnkfelix",
     "rylev",
+    "sophiajt",
```

Comment:
Looking at the history it seems like this removal was an accident (removed instead of archived).